bob-base/bob-base.conf

```diff
@@ -0,0 +1,53 @@
+[Build]
+Environment=KERNEL_CONFIG_SNIPPETS=bob-base/kernel.config
+WithNetwork=true
+
+[Content]
+ExtraTrees=bob-base/mkosi.extra
+PostInstallationScripts=bob-base/mkosi.postinst
+BuildScripts=bob-base/mkosi.build
+
+Packages=podman
+         runc
+         dropbear
+         socat
+         iptables
+         iproute2
+         conntrack
+         netfilter-persistent
+         openntpd
+         curl
+         jq
+         logrotate
+         sudo
+         uidmap
+         passt
+         fuse-overlayfs
+         libdevmapper1.02.1
+         libjson-c5
+         openssh-sftp-server
+         udev
+         libsnappy1v5
+
+# TODO: some of those could be removed
+BuildPackages=build-essential
+              git
+              gcc
+              zlib1g-dev
+              libzstd-dev
+              libleveldb-dev
+              libsnappy-dev
+              libpq-dev
+              libssl-dev
```

Comment:
This group of lib*-dev dependencies are the lighthouse-specific ones. iirc, everything else is for other deps